core/cloudflow-crd/src/main/scala/akka/datap/crd/App.scala

```diff
@@ -73,13 +84,9 @@ object App {
       .withPlural(Plural)
       .withShortNames(Short)
       .endNames()
-      .withVersion(GroupVersion)
+      .withVersions(customResourceDefinitionVersion)
       .withScope("Namespaced")
       .withPreserveUnknownFields(true)
-      .withNewSubresources()
```

Comment:
all these seem to be part of `CustomResourceDefinitionVersion`/ `CustomResourceDefinitionVersionBuilder` now